x/poolmanager/msg_server_test.go

```diff
@@ -80,6 +80,10 @@ func (s *KeeperTestSuite) TestSplitRouteSwapExactAmountIn() {
 			s.Setup()
 			ctx := s.Ctx
 
+			poolManagerParams := s.App.PoolManagerKeeper.GetParams(ctx)
+			poolManagerParams.TakerFeeParams.DefaultTakerFee = sdk.MustNewDecFromStr("0.01")
+			s.App.PoolManagerKeeper.SetParams(ctx, poolManagerParams)
+
```

Comment:
These needed to be added due to the && that was added to the taker fee logic. This test before was going through the loop with a taker fee of 0%, so now if it isn't set we just skip this logic.